src/commands/deleteserver.js

```diff
@@ -0,0 +1,139 @@
+const {
+    SlashCommandBuilder,
+    PermissionsBitField,
+    SlashCommandStringOption,
+    ActionRowBuilder,
+    ButtonBuilder,
+    ButtonStyle,
+} = require("discord.js");
+const config = require("../config");
+const Command = require("../structures/command");
+
+class DeleteServerCommand extends Command {
+    constructor() {
+        super({
+            active: true,
+            data: new SlashCommandBuilder()
+                .setName("deleteserver")
+                .setDescription("Delete um servidor da database do agent.")
+                .setDMPermission(false)
+                .setDefaultMemberPermissions(
+                    PermissionsBitField.Flags.ManageGuild
```

Comment:
pra guards poderem usar
```suggestion
                    PermissionsBitField.Flags.ManageRoles
```